Plugins/GitUIPluginInterfaces/ISetting.cs

```diff
@@ -37,6 +37,7 @@ public interface ISettingControlBinding
         /// Saves value from Control to settings
         /// </summary>
         /// <param name="settings"></param>
+        /// <param name="areSettingsEffective"></param>
```

Comment:
💭 Honestly I'd almost rather remove the previous empty comment than add another useless one. My normal approach though is the one I've stated a couple times previously - disable CS1573.